docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:latest
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+COPY package.json /usr/src/app/
+RUN npm install
+COPY . /usr/src/app
+EXPOSE 3000
+CMD [ "./utils/wait-for-it.sh", "mongo:27017", "--", "npm", "start" ]
```

Comment:
`utils` folder needs to be moved here, as it's looking for the relative path.

Reply:
The `CMD` here is running relative to the root of the repository.  
`COPY` is copying everything from the root, then `./utils` is referenced from there